src/main/java/blitz/earthquake/json/EarthquakeFrame.java

```diff
@@ -0,0 +1,99 @@
+        setTitle("EarthquakeFrame");
+        setSize(300, 600);
+        setDefaultCloseOperation(EXIT_ON_CLOSE);
+
+        setLayout(new BorderLayout());
+
+        JPanel radioButtonPanel = new JPanel(new FlowLayout(FlowLayout.LEFT));
+        radioButtonPanel.add(oneHour);
+        radioButtonPanel.add(sigMonth);
+
+        ButtonGroup group = new ButtonGroup();
+        group.add(oneHour);
+        group.add(sigMonth);
+
+        add(radioButtonPanel, BorderLayout.PAGE_START);
+        add(jlist, BorderLayout.CENTER);
+
+        EarthquakeService service = new EarthquakeServiceFactory().getService();
+
+        oneHour.addActionListener(e -> fetchData(service.oneHour()));
+        sigMonth.addActionListener(e -> fetchData(service.sigMonth()));
+
+        jlist.setSelectionMode(ListSelectionModel.SINGLE_SELECTION);
+        jlist.setSelectionMode(ListSelectionModel.SINGLE_SELECTION);
+        jlist.addListSelectionListener(e -> {
+            if (!e.getValueIsAdjusting()) {
+                String selectedValue = jlist.getSelectedValue();
+                if (selectedValue != null) {
+                    String[] parts = selectedValue.split(" ");
+                    double longitude = Double.parseDouble(parts[parts.length - 2]);
+                    double latitude = Double.parseDouble(parts[parts.length - 1]);
+                    if (Desktop.isDesktopSupported()
+                            && Desktop.getDesktop().isSupported(Desktop.Action.BROWSE)) {
+                        try {
+                            Desktop.getDesktop().browse(new URI(
+                                    "https://www.google.com/maps/search/?api=1&query="
+                                            + latitude + "," + longitude));
+                        } catch (IOException | URISyntaxException ex) {
+                            throw new RuntimeException(ex);
+                        }
+                    }
+                }
+            }
+        });
+
+                fetchData(service.oneHour());
+            }
+
+    private void fetchData(Single<FeatureCollection> single) {
+        if (disposable != null && !disposable.isDisposed()) {
+            disposable.dispose();
+        }
+        disposable = single
+                .subscribeOn(Schedulers.io())
+                .observeOn(SwingSchedulers.edt())
+                .subscribe(
+                        this::handleResponse,
+                        Throwable::printStackTrace
+                );
+    }
+
+    private void handleResponse(FeatureCollection response) {
+        String[] listData = new String[response.features.length];
+        for (int i = 0; i < response.features.length; i++) {
+            Feature feature = response.features[i];
+            double lat = feature.geometry.coordinates[0];
+            double lng = feature.geometry.coordinates[1];
+            listData[i] = feature.properties.mag + " " + feature.properties.place + " " + lat + " " + lng;
```

Comment:
rather than putting the lat/lon into the String, save the FeatureCollection and use it later.